protocol/dydx-cosmwasm/src/msg.rs

```diff
@@ -43,61 +35,42 @@ pub enum OrderConditionType {
   TakeProfit = 2,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
-pub struct OrderId {
-  pub subaccount_id: SubaccountId,
-  pub client_id: u32,
-  pub order_flags: u32,
-  pub clob_pair_id: u32,
-}
-
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
-pub struct Order {
-  pub order_id: OrderId,
-  pub side: OrderSide,
-  pub quantums: u64,
-  pub subticks: u64,
-  pub good_til_oneof: GoodTilOneof,
-  pub time_in_force: OrderTimeInForce,
-  pub reduce_only: bool,
-  pub client_metadata: u32,
-  pub condition_type: OrderConditionType,
-  pub conditional_order_trigger_subticks: u64,
-}
-
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
-#[serde(rename_all = "snake_case")]
-pub enum GoodTilOneof {
-    GoodTilBlock(u32),
-    GoodTilBlockTime(u32),
-}
-
-
 #[non_exhaustive]
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum DydxMsg {
```

Comment:
Here or wherever else better, could we add some documentation for these messages? This should call out each message is a 1:1 mapping to relevant protocol side message, without `sender` which is by default the contract itself.

Reply:
Good callout, will do